MANUAL_REGRESSION.md

```diff
@@ -112,7 +112,40 @@ To check if a device&browser supports getUserMedia you can check this link - [ca
     - user should see `Something's gone wrong` error
     - user should persist on the same screen
 
-##### 8. Check happy path flow on other desktop browsers
+##### 8. Prevent to open cross-device URL on web browsers.
+    (on private mode of: Google Chrome, Firefox, Safari and Microsoft Edge browsers)
+
+0. Given user is on first page of cross-device flow
+1. Click on the `Copy` button
+2. Open a new tab of the browser
+3. Paste the link
+    - user should see `Something's gone wrong` message
+    - user should see `You must open this link on a mobile device` message
+    - user should see the icon with the phone, screen and the red cross
+
+##### 9. Prevent upload fallback when requested
```

Comment:
This can only be tested if the surge link has the production build configuration. I will open a ticket to refactor the build config so that all builds will have prod config